config/management/src/lib.rs

```diff
@@ -318,6 +425,35 @@ pub mod tests {
         command.owner_key()
     }
 
+    fn validator_config(
+        owner_address: AccountAddress,
+        validator_address: Multiaddr,
+        fullnode_address: Multiaddr,
+        namespace: &str,
+    ) -> Transaction {
+        let args = format!(
+            "
+                validator_config
+                validator-config
```

Comment:
the double validator-config is confusing, sth like register_validator?

Reply:
that is the binary name, it was from an earlier version before I started calling it management... sometimes this happens